app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :increment_no_of_likes
+
+  private
+
+  def increment_no_of_likes
+    post.update(likes_counter: post.likes_counter + 1)
+  end
```

Comment:
- [ ] Kindly make refrenece to my comments on the `Post` model regarding the use of `increment` 👍

Reply:
Okay I got it! Let me Implement it.